src/landing/download_item.rs

```diff
@@ -197,9 +197,23 @@ live_design! {
             color: #fff,
         }
 
-        <Information> {}
-        <Progress> {}
-        <Actions> {}
+        <View> {
+            width: Fill,
+            height: Fit,
+            <Information> {}
+        }
+
+        <View> {
+            width: Fill,
+            height: Fit,
+            <Progress> {}
+        }
+
+        <View> {
+            width: Fill,
+            height: Fit,
+            <Actions> {}
+        }
```

Comment:
While this works we can clean up a bit:
It seems that you can actually achieve this by simply changing the `width` and `height` property on those elements directly, instead of wrapping them in views.

You can keep:
```rs
<Information> {}
<Progress> {}
<Actions> {}
```

And update those values directly on each element:
```rs
Actions = <View> {
 width: Fill, height: Fit
// ...
}
```

Reply:
OK, I got it,thanks your reviewing.

Reply:
Thanks!